docs/Feature Documentation/Audio and Video Streaming Setup/index.md

```diff
@@ -89,9 +54,6 @@ VideoStreamConsumer = pipe
 AudioStreamConsumer = pipe
 ```
 
-### HMI Modifications
-Using pipe streaming may require a modification to the SDL HMI.
-
 #### VIDEO
```

Comment:
This subheading should be removed since video socket streaming doesn't have a similar heading and the `####` subheadings are already used for the video/audio format

Reply:
This header applies to both socket and pipe streaming as the description under it says, I think it makes a lot of sense to preface the VIDEO and AUDIO sections with a header that includes them both, called something like HMI modifications.

I moved the whole section under SDL HMI setup which should help the confusion that it just applies to pipe streaming. Let me know what you think.

Reply:
Sorry should have been more specific I was refering to the `#### VIDEO` heading

Currently the pipe and socket Streaming sections are setup as 
```md
## <Pipe/Socket> Streaming

### Configuration (smartDeviceLink.ini)
...
### Video Stream <Pipe/Socket>
...
### Audio Stream <Pipe/Socket>
...
```
There is however an extra subheading above the `### Video Stream Pipe` section which is somewhat out of place
![Screen Shot 2020-03-04 at 3 14 51 PM](https://user-images.githubusercontent.com/10703941/75919565-c92d1100-5e2b-11ea-9e0b-1b97f4d83ffd.png). I was saying that we should remove https://github.com/smartdevicelink/sdl_core_guides/pull/71/commits/b07a1990e1799725a09996753b1db4bba73b664b#diff-e3759fc012a839fcb83e8cf8b6408a10R65

Reply:
AH! great catch. fixed in https://github.com/smartdevicelink/sdl_core_guides/pull/71/commits/f7e530b4c9121bf3adb5c693975b2aa2d0962751